Pengumpulan Tugas

kpratama24 opened the pull request.

Commit:
Commenting

Add "Converting to C#"

Commit:
Rename LanguagesExploration.java to LanguagesExploration.cs

Commit:
Update LanguagesExploration.cs

initialize namespace, static void main

Commit:
Update LanguagesExploration.cs

setter getter implementation for cs (acccording to Microsoft Visual Studio code design guidelines)

farhanfazrian committed:
Update LanguagesExploration.cs

Fachrann committed:
Update LanguagesExploration.cs

valentinancy committed:
Update LanguagesExploration.cs

getter and setter according to MSDN code guidelines

cklsh committed:
Update LanguagesExploration.cs

wahyuuwewe committed:
Update LanguagesExploration.cs

wahyuuwewe committed:
Update LanguagesExploration.cs

pascalalfadian commented:
Dear @kpratama24 

Terima kasih. Mohon perbaiki dua hal:
1. Kembalikan file `LanguagesExploration.java` (jangan dihapus), karena akan di merge dengan repo utama
2. Masih salah di nomer 23 (diharapkan 26, yang keluar 13). Sepertinya kamu salah cara _override_ nya.

Best regards;
Pascal

Commit:
Create LanguagesExploration.java

Commit:
Update LanguagesExploration.cs

Overriding method using override modifier in derived class and virtual modifier in base class

kpratama24 commented:
Sudah diperbaiki pa. Lupa ditambahkan virtual modifier dan override modifiernya
Terima kasih

pascalalfadian commented:
:+1: thanks
